ci/pipelines/deploy-to-test.yml

```diff
@@ -5214,6 +5223,102 @@ jobs:
         icon_emoji: ":concourse:"
         username: pay-concourse
 
+  - name: deploy-webhooks-egress
+    serial: true
+    serial_groups: [deploy-application]
+    plan:
+      - get: webhooks-egress-ecr-registry-test
+        trigger: true
+      - get: pay-infra
+      - get: pay-ci
+      - load_var: application_image_tag
+        file: webhooks-egress-ecr-registry-test/tag
+      - task: create-notification-snippets
+        file: pay-ci/ci/tasks/create-egress-notification-snippets.yml
+        params:
+          ACTION_NAME: Deployment
+          APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+          ENV: test-12
+      - load_var: failure_snippet
+        file: snippet/failure
+      - load_var: success_snippet
+        file: snippet/success  
+      - task: assume-role
+        file: pay-ci/ci/tasks/assume-role.yml
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+          AWS_ROLE_SESSION_NAME: terraform-test-assume-role
+      - load_var: role
+        file: assume-role/assume-role.json
+        format: json
+      - task: check-release-versions
+        file: pay-ci/ci/tasks/check-release-versions.yml
+        params:
+          <<: *aws_assumed_role_creds
+          AWS_REGION: "eu-west-1"
+          CLUSTER_NAME: "test-12-fargate"
+          APP_NAME: webhooks-egress
+          APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+      - task: deploy-to-test
+        file: pay-ci/ci/tasks/deploy-webhooks-egress.yml
+        params:
+          <<: *aws_assumed_role_creds
+          APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+          ACCOUNT: test
+          ENVIRONMENT: test-12
+      - task: wait-for-deploy
+        file: pay-ci/ci/tasks/wait-for-deploy.yml
+        params:
+          APP_NAME: webhooks-egress
+          APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+          <<: *aws_assumed_role_creds
+          ENVIRONMENT: test-12
+    <<: *put_success_slack_notification      
+    <<: *put_failure_slack_notification
+  
+  - name: smoke-test-webhooks-egress
```

Comment:
There's no smoke tests that interact with webhooks at all right? I don't think you should have the smoke test task unless it's usefully covering the thing being released

Reply:
Chatted with Steven, there will be smoke tests in the future so keep this 👍